ethereum/contracts/TablelandRigs.sol

```diff
@@ -54,16 +70,19 @@ contract TablelandRigs is
     // URI for contract info.
     string private _contractInfoURI;
 
-    // Table prefix for the Rigs pilot sessions table.
-    string private constant PILOT_SESSIONS_PREFIX = "pilot_sessions";
-
     // Table ID for the Rigs pilot sessions table.
     uint256 private _pilotSessionsTableId;
 
-    // Tracks the Rig `tokenId` to its current `Pilot`.
-    mapping(uint16 => Pilot) internal _pilots;
+    // Tracks the Rig `tokenId` to its current pilot, represented as a packed `uint256`.
+    //
+    // Bits layout:
+    // - [0..63]    `startTime` - starting block number of pilot's flight time
+    // - [64..95]   `pilotId` - ERC-721 token ID of the pilot at `pilotContract`
+    // - [96..255]  `pilotContract` - address of the ERC-721 contract for the pilot
+    mapping(uint16 => uint256) internal _pilots;
```

Comment:
@sanderpick i was hacking around with packing pilot data into a `uint256`, instead of representing pilots as a `Pilot` struct (which contained the start time and a packed `uint192` variable of pilot ID and contract). i based it off of the ERC721A impl.

it didn't really have a significant gas savings, so i'm curious if you think it's worthwhile to use this impl, or this all hurts readability.